From 28922d4ed0a380d26a8634e8201fd1a001c0db94 Mon Sep 17 00:00:00 2001
From: Sanket Diwale <diwales@mit.edu>
Date: Thu, 12 May 2022 19:09:51 -0400
Subject: [PATCH 2/2] update addcompilerRT.cmake for windows

---
 compiler-rt/cmake/Modules/AddCompilerRT.cmake | 1 +
 1 file changed, 1 insertion(+)

diff --git a/compiler-rt/cmake/Modules/AddCompilerRT.cmake b/compiler-rt/cmake/Modules/AddCompilerRT.cmake
index bc69ec95c4..c21981e569 100644
--- a/compiler-rt/cmake/Modules/AddCompilerRT.cmake
+++ b/compiler-rt/cmake/Modules/AddCompilerRT.cmake
@@ -10,6 +10,7 @@ function(set_target_output_directories target output_dir)
   # RUNTIME_OUTPUT_DIRECTORY_DEBUG, RUNTIME_OUTPUT_DIRECTORY_RELEASE, ...
   if(CMAKE_CONFIGURATION_TYPES)
     foreach(build_mode ${CMAKE_CONFIGURATION_TYPES})
+      string(REGEX REPLACE "\\$\\(Configuration\\)" "$<CONFIG>" output_dir "${output_dir}")
       string(TOUPPER "${build_mode}" CONFIG_SUFFIX)
       set_target_properties("${target}" PROPERTIES
           "ARCHIVE_OUTPUT_DIRECTORY_${CONFIG_SUFFIX}" ${output_dir}
-- 
2.30.0.windows.1

